homeassistant/components/qwikswitch.py

```diff
@@ -33,7 +33,10 @@
         vol.Required(CONF_URL, default='http://127.0.0.1:2020'):
             vol.Coerce(str),
         vol.Optional(CONF_DIMMER_ADJUST, default=1): CV_DIM_VALUE,
-        vol.Optional(CONF_BUTTON_EVENTS): vol.Coerce(str)
+        vol.Optional(CONF_BUTTON_EVENTS): cv.ensure_list_csv,
```

Comment:
`ensure_list_csv` should only be used to avoid breaking changes. It's better to use plain `ensure_list` if possible.

Reply:
Does it work to add a default empty list here, so you don't need to add that in `setup`?

Reply:
It will work, only reason I dont like it is that validating then adds this empty key to the config (which really is not used by many people). 

`ensure_list_csv` is indeed for backward compatibility. It used to be a text deliminated string